src/DfE.ExternalApplications.Web/appsettings.Development.json

```diff
@@ -28,7 +28,7 @@
     ]
   },
   "ExternalApplicationsApiClient": {
-    "BaseUrl": "https://s184d01-rsd-frontdoor-extapp-api-g5btf4dfgzhzcvc2.a03.azurefd.net/",
+    "BaseUrl": "https://localhost:7089/",
```

Comment:
Please revert this otherwise the Dev environment will break, you need to make sure the appsettings.Development.json is untouched.

Reply:
I think @jack-nimbleapproach said I should include the changes of these 2 files. Is it because it's branched off another branch rather than the main branch?